partials/forms/select.html

```diff
@@ -1,12 +1,26 @@
-<div id="{{id}}-group" class="form-group{{#compound}} form-group-compound{{/compound}}{{#error}} error{{/error}}">
+<div id="{{id}}-group" class="form-group{{#compound}} form-group-compound{{/compound}}
+{{#error}} error{{/error}}
+{{#groupClassName}}
+    {{groupClassName}}
+{{/groupClassName}}"
```

Comment:
I think it's worth matching the existing pattern here for custom class names using the `className` option.